woocommerce/woocommerce-gutenberg-products-block
# Mini Cart Block: Remove not necessary div #5419

This PR removes a not necessary div to prevent not necessary nested DOM structure.





Fixes #5419


### Testing

### Manual Testing

How to test the changes in this Pull Request:

Check out this branch:

1. Using Storefront, add the Mini Cart block as widget.
2. View the page and inspect elements using the browser's dev tools.
3. Click on the mini-cart to open the mini-cart-contents drawer.
4. Find the `wc-block-mini-cart__template-part` class element.
5. Check that the direct child is a div with `wp-block-woocommerce-mini-cart-contents` class element.
